rust/kcl-lib/src/execution/typed_path.rs

```diff
@@ -150,6 +150,23 @@ impl TypedPath {
             self.0.file_name().and_then(|s| s.to_str()).map(|s| s.to_string())
         }
     }
+
+    /// Extracts the stem (non-extension) portion of [`self.file_name`].
+    #[must_use]
+    pub fn file_stem(&self) -> Option<String> {
```

Comment:
What's with this part? Doesn't seem to be used?

Reply:
Yeah, I changed something to reuse other code. You can delete this.